5.7/ubuntu/22.04/Dockerfile

```
@@ -0,0 +1,42 @@
+FROM ubuntu:22.04
+
+LABEL maintainer="SwiftWasm Maintainers <hello@swiftwasm.org>"
+LABEL Description="Docker Container for the SwiftWasm toolchain and SDK"
+LABEL org.opencontainers.image.source https://github.com/swiftwasm/swiftwasm-docker
+
+RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && apt-get -q update && \
+    apt-get -q install -y \
+    binutils \
+    git \
+    gnupg2 \
+    libc6-dev \
+    libcurl4 \
+    libedit2 \
+    libgcc-9-dev \
+    libpython2.7 \
+    libsqlite3-0 \
+    libstdc++-9-dev \
+    libxml2 \
+    libz3-dev \
+    pkg-config \
+    tzdata \
+    zlib1g-dev \
+    && rm -r /var/lib/apt/lists/*
+
+
+ARG SWIFT_TAG=swift-wasm-5.7.3-RELEASE
+ENV SWIFT_PLATFORM_SUFFIX=ubuntu20.04_x86_64.tar.gz
```

I just uploaded focal build for 5.7.3
```suggestion
ENV SWIFT_PLATFORM_SUFFIX=ubuntu22.04_x86_64.tar.gz
```